apps/houstonmetro/houstonmetro.star

```diff
@@ -190,25 +204,27 @@ def truncate_location(full_string):
 def get_stations(location):
     loc = json.decode(location)
     coordinates = truncate_location(str(loc["lat"])) + "|" + truncate_location(str(loc["lng"]))
-    key = SUBSCRIPTION_KEY or ""
-    location_cache = cache.get(ROUTE_INFO_CACHE_KEY + coordinates)
-    if location_cache:
-        response = location_cache
-    else:
-        location_endpoint = "https://houstonmetro.azure-api.net/data/GeoAreas('" + coordinates + "|.5')/Stops?subscription-key=" + key
-        response = http.get(location_endpoint)
-
-        # TODO: Determine if this cache call can be converted to the new HTTP cache.
-        cache.set(ROUTE_INFO_CACHE_KEY + coordinates, response.body(), ROUTE_INFO_CACHE_TTL)
+    key = SUBSCRIPTION_KEY
     stops = []
-    if response.json()["value"]:
-        for station in response.json()["value"]:
-            stops.append(
-                schema.Option(
-                    display = station["Name"],
-                    value = station["StopId"],
-                ),
-            )
+    if key:
+        location_cache = cache.get(ROUTE_INFO_CACHE_KEY + coordinates)
```

Comment:
If you haven't already seen it, check out the http module's new(ish) cache functionality. Could simplify the code here a bit, and also future proof against future changes to the scope of the cache module (we're thinking of making it per-app-installation instead of per-app).

Reply:
More about that over here: https://discuss.tidbyt.com/t/http-caching-client/5129